doc/news/changes/minor/20210223Fehling

```diff
@@ -1,4 +1,4 @@
-New: Member function DoFHandler::prepare_coarsening_and_refinement()
+New: Function hp::Refinement::limit_plevel_difference()
```

Comment:
```
New: Function hp::Refinement::limit_p_level_difference()
```
?

Are the other functions with similar names (h/p) in the library?

Reply:
There are a few in the `hp::Refinement` namespace, like `choose_p_over_h`.

`prepare_c_and_r` was simply wrong in this context as it is not a mandatory preparation. The function should rather describe what it's doing.